Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
@omeshp OK, I've also added the locator to the LatestBuild endpoint. But IMHO the extension as it is now is not usable in an autotrigger scenario with multiple artifacts, as it will choose a build from a random branch.